scss/tokens/_raw.scss

```diff
@@ -17,3 +17,130 @@ $ouds-opacity-700: .64 !default;
 // $ouds-opacity-800: .88 !default;
 $ouds-opacity-900: 1 !default;
 // scss-docs-end ouds-opacities-variables
+
+// Elevation
+
+// scss-docs-start ouds-raw-zindex
```

Comment:
These tokens won't be generated. We can keep raw values directly in `_variables.scss` (or anywhere else).